apps/election-manager/src/screens/TallyReportScreen.tsx

```diff
@@ -36,18 +39,63 @@ const TallyHeader = styled.div`
   }
 `
 
+const MetadataSection = styled.div`
+  h3 {
+    margin-bottom: 0.25em;
+  }
+`
+
```

Comment:
This is not necessary if you replace the `<div>` with a `<React.Fragment>`. Prose will handle all spacing between typographical elements.